compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIESplitLogicalObjFifos.cpp

```diff
@@ -215,8 +219,9 @@ void AMDAIESplitLogicalObjFifosPass::runOnOperation() {
 
 }  // namespace
 
-std::unique_ptr<Pass> createAMDAIESplitLogicalObjFifosPass() {
-  return std::make_unique<AMDAIESplitLogicalObjFifosPass>();
+std::unique_ptr<Pass> createAMDAIESplitLogicalObjFifosPass(
+    AMDAIESplitLogicalObjFifosOptions options) {
```

Comment:
Instead of providing `numCols` as an option, I think we can retrieve this information from the target attribute instead. I have this change in a follow-up.